src/main/java/org/opentripplanner/middleware/controllers/api/ApiController.java

```diff
@@ -119,7 +120,7 @@ protected void buildEndpoint(ApiEndpoint baseEndpoint) {
                     // Note: there exists a method withResponseAsCollection, but unlike what its name suggests,
                     // it does exactly the same as .withResponseType and does not generate a return type array.
                     // See issue https://github.com/manusant/spark-swagger/issues/12.
-                    .withResponseType(Array.newInstance(clazz, 0).getClass()),
+                    .withResponseType(ResponseList.class),
```

Comment:
Need to add the spark-swagger code to document the new query parameters `limit` and `page`.